modulemd/modulemd-foreman-el8.yaml

```diff
@@ -26,3 +26,24 @@ data:
         platform: ['el8']
         ruby: ['2.7']
         postgresql: ['12']
+...
+---
+document: modulemd
+version: 2
+data:
+  name: foreman-devel
```

Comment:
I picked `foreman-devel` over `foreman-build` as it's useful for non-build things too, but the name really does not matter much.